docs/pages/blog/turbo-1-4-0.mdx

```diff
@@ -0,0 +1,126 @@
+
+Turborepo 1.4 improves usage with frameworks and adds new examples:
+
+- [**Automatic environment variable inclusion:**](#automatic-environment-variable-inclusion): We'll automatically infer popular framework environment variables for you. No need to declare them yourself in `turbo.json`.
+- [**`eslint-plugin-turbo`**](#eslint-plugin-turbo): Enhanced feedback with a new ESLint plugin
+- [**New framework and library examples:**](#new-framework-and-library-examples) New starters and examples requested by the community.
+
+Update today by running `npm install turbo@latest`.
+
+## Automatic environment variable inclusion
+
+To help ensure correct caching across environments, Turborepo will now automatically infer and include public environment variables when calculating cache keys for apps built with Astro, Create React App, Gatsby, Next.js, Nuxt, SvelteKit, Vite, Vue, and more. You can safely remove these framework-specific public environment variables from `turbo.json` if you have manually declared them.
+
+```diff filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+        "^build"
+-       // Include build time public inlined environment variables that
+-       // are different in development and production, so that
+-       // Turborepo does not use the same cached build
+-       // accross environments
+-       "$NEXT_PUBLIC_EXAMPLE_ENV_VAR"
+      ]
+    }
+  }
+}
+```
+
+Note that this automatic detection and inclusion only works if Turborepo successfully infers the framework your apps are built with. Additionally, environment variables will only be part of the cache key in the workspaces where that framework is used. In other words, environment variables inferred for Next.js apps, will only apply to the cache key for workspaces inferred as Next.js apps. Other workspaces in the monorepo will not be impacted.
+
+For example, consider a monorepo with a three workspaces: a Next.js project, a Create React App project, both depending on a shared Typescript package and each with their own`build` script. Let's say that this Turborepo has a standard `turbo.json` pipeline that builds them all in order:
+
+```jsonc filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+				"^build",
+			]
+    }
+}
+```
+
+As of 1.4, when you run `turbo run build`, Turborepo will not consider any build time environment variables relevant when building the TypeScript packages. However, when building the Next.js app, Turborepo will infer that environment variables starting with `NEXT_PUBLIC_` could alter the output of the `.next` folder and should thus be included when calculating the hash. Similarly, when calculating the hash of the Create React App's `build` script, all build time environment variables starting with `REACT_APP_PUBLIC_` will be included.
+
+This improvement in hash specificity by framework is a significant step toward optimal, safe, and correct caching.
+
+## `eslint-plugin-turbo`
```

Comment:
if we recommend `eslint-config-turbo`, should the heading be config instead of plugin?